utils/src/tree/v2/ActivityChildItem.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { TreeItem, TreeItemCollapsibleState } from "vscode";
+import { TreeItemIconPath } from "../../..";
+import * as types from '../../../index';
+
+export class ActivityChildItem implements types.ActivityChildItemBase {
```

Comment:
Shouldn't this have a `getChildren` method to fully implement `types.ActivityChildItemBase`? Otherwise did you mean for it to be `abstract`?

Reply:
Oh I meant for `getChildren` to be optional